edgelet/contrib/debian/preinst

```diff
@@ -12,6 +12,11 @@ add_groups()
     fi
     mkdir -p /var/lib/aziot/edged
 
+    # Create an edgeagentuser and add it to iotedge group
+    if ! getent passwd edgeagentuser >/dev/null; then
+        useradd -g iotedge -c "edgeAgent user" -ms /bin/bash -u 13622 edgeagentuser
```

Comment:
Same here. Login shell should be `/sbin/nologin`

Reply:
fixed